feeds/wlan-ap/opensync/src/platform/openwrt/src/rrm/src/rrm_channel.c

```diff
@@ -19,25 +19,32 @@ static struct ev_timer  rrm_nf_timer;
 static void rrm_nf_timer_handler(struct ev_loop *loop, ev_timer *timer, int revents);
 
 enum {
-	WIF_ATTR_BEACON_RATE,
 	WIF_ATTR_PROBE_ACCEPT_RATE,
+	WIF_ATTR_CLIENT_CONNECT_THRESHOLD,
+	WIF_ATTR_CLIENT_DISCONNECT_THRESHOLD,
 	__WIF_ATTR_MAX,
 };
 
 static const struct blobmsg_policy wifi_config_policy[__WIF_ATTR_MAX] = {
-		[WIF_ATTR_BEACON_RATE] = { .name = "beacon_rate", .type = BLOBMSG_TYPE_INT32 },
-		[WIF_ATTR_PROBE_ACCEPT_RATE] = { .name = "rssi_reject_assoc_rssi", .type = BLOBMSG_TYPE_INT32 },
+		[WIF_ATTR_PROBE_ACCEPT_RATE] = { .name = "rssi_ignore_probe_request", .type = BLOBMSG_TYPE_INT32 },
+		[WIF_ATTR_CLIENT_CONNECT_THRESHOLD] = { .name = "signal_connect", .type = BLOBMSG_TYPE_INT32 },
+		[WIF_ATTR_CLIENT_DISCONNECT_THRESHOLD] = { .name = "signal_stay", .type = BLOBMSG_TYPE_INT32 },
 };
 
 const struct uci_blob_param_list wifi_config_param = {
 		.n_params = __WIF_ATTR_MAX,
 		.params = wifi_config_policy,
 };
 
-static void rrm_set_basic_rate(const char *if_name, int32_t rate)
+static void rrm_set_mgmt_rate(const char *if_name, int32_t rate)
```

Comment:
please remove this stub from the PR

Reply:
This is something to be taken up soon. So did set it as a TODO. Nevertheless, Can remove it for now.

Reply:
> please remove this stub from the PR

Done